addons/web/static/src/search/favorite_menu/favorite_menu.xml

```diff
@@ -8,7 +8,7 @@
                 <span class="o_dropdown_title">Favorites</span>
             </t>
             <t t-set="currentGroup" t-value="null"/>
-            <t t-foreach="items" t-as="item" t-key="item.id or item.key">
+            <t t-foreach="this.items" t-as="item" t-key="item.id or item.key">
```

Comment:
`items` is no longer in the `ctx` of the template here ? or is it now different from the original one ?

Reply:
no, just the getter was not bound on `this` (and by doing this.items I force the getter to be bounded)